bus-mapping/src/circuit_input_builder/input_state_ref.rs

```diff
@@ -318,7 +318,7 @@ impl<'a> CircuitInputStateRef<'a> {
         // Perform the write to the account in the StateDB
         if matches!(rw, RW::WRITE) {
             match op.field {
-                AccountField::Nonce => account.nonce = op.value,
+                AccountField::Nonce => account.nonce = op.value.low_u64(),
```

Comment:
```suggestion
                AccountField::Nonce => account.nonce = op.value.as_u64(),
```
With `as_u64` we get a panic in case the value doesn't fit in u64.  I think this can be useful to avoid silent bugs.

Reply:
Addressed in [0fc8072](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1409/commits/0fc80723760ada93d2128cae58c6e19a09e8ba62)